### PR TITLE
XW-4953 | wds-avatar-stack component

### PR DESCRIPTION
**Pull request by rybmat**
https://wikia-inc.atlassian.net/browse/XW-4953

Ember component for avatar-stack

@Wikia/x-wing

### CODE REVIEW

File: app/components/wds-avatar-stack.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import Component from '@ember/component';
+import {computed} from '@ember/object';
+
+
+export default Component.extend({
+	classNames: ['wds-avatar-stack'],
+	overflow: computed('avatars', function() {
+		const count = this.get('avatars').length;
+
+		return count > 5 ? this.get('avatars').length - 5 : 0;
```
Comment (hakubo) [inline app/components/wds-avatar-stack.js]: this 5 could be a property, so it can be overriden if needed

Comment (hakubo) [reply app/components/wds-avatar-stack.js]: count - 5?

File: app/components/wds-avatar-stack.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import Component from '@ember/component';
+import {computed} from '@ember/object';
+
+
+export default Component.extend({
+	classNames: ['wds-avatar-stack'],
```
Comment (hakubo) [inline app/components/wds-avatar-stack.js]: avatars could be a property on the compent as a null
so it still blows when you don't pass it
but this way when you look at the component you see what properties it have

File: app/components/wds-avatar-stack.js (hunk 1 above)
Comment (rybmat) [reply app/components/wds-avatar-stack.js]: @hakubo I don't think we can allow for more elements than 5 unless we'll figure out how to handle it better in scss: https://github.com/Wikia/design-system/blob/3e6f170b7379a523f3ce6c5def7db03bab063405/dist/scss/wds-components/_avatar-stack.scss#L10

Comment (latata) [reply app/components/wds-avatar-stack.js]: I would extract it to the property and leave a comment that scss file should be updated together with this value.

File: app/components/wds-avatar-stack.js (hunk 2 above)
Comment (rybmat) [reply app/components/wds-avatar-stack.js]: done

**Pull request merged**